packages/status-react/src/routes/index.tsx

```diff
@@ -62,8 +76,20 @@ export const Community = (props: Props) => {
               <MainSidebar />
               <Routes>
                 <Route
-                  path="/:id"
+                  path="/"
+                  element={
+                    <RootGate>
+                      {/* todo?: empty state/page */}
+                      {/* todo?: navbar at least; think community w/o chats yet, but already has members to inspect */}
```

Comment:
follow-up: discuss:

Reply:
AFAIK, Community should always have at least one chat.